damnit/gui/main_window.py

```diff
@@ -971,7 +980,7 @@ def test_context(self):
         else:
             # Move the error widget down
             height_unit = self.height() // 7
-            self._editor_parent_widget.setSizes([6 * height_unit, height_unit])
```

Comment:
Should the denominator in the line above be changed to 6? It looks like we're aiming for 1/6 (previously 1/7) the total height.

Reply:
No that's the right number, the idea is that we divide the height of the window into 7 `height_unit`'s, and give 6/7 to the editor and 1/7 to the error widget (the second argument could be more clearly written as `1 * height_unit`).

Reply:
I think I made this confusing by commenting on the 'before' side of the diff rather than the 'after'. The code in this PR appears to be giving 5/7 to the editor and 1/7 to the error widget. I think Qt shares out the extra 1/7 between the widgets, though, so it doesn't matter in practice.

Reply:
Ah yes I missed that, you're quite right :facepalm: I'll make a lil' fix.